### ISSUE TITLE
CTM texture not working well

### ISSUE THREAD

**Issue by fr1g** [Chocohead/OptiFabric#1149]
1.20.1-optifine I5, optifabric 1.13.25
I'm checking on the stained glass panel and found that if here's a light nearby then the glass will be in a mess. Turning off the connection texture it will be fine. In the pure optifine client has no such issue. Tested BSL and Sildur Vibr. found it's not caused by shaders.
![image](https://github.com/Chocohead/OptiFabric/assets/53085628/b7fd47d7-20b0-4a82-8a57-55bdedf8a6a4)
just like this ^

**Comment by fr1g** [comment]
PS:
I found that this will happen on west AND north surfaces of glass panel, nothing to do with lightning.

**Comment by RU-B0** [comment]
Just wanted to comment how I fixed this issue: I found out that a mod called fusion (https://modrinth.com/mod/fusion-connected-textures) launches with OptiFine just fine. Using Midnighttigger's Default Connected Textures (https://modrinth.com/resourcepack/mt-ct-d/version/tfKvt8nl) and putting it as lowest priority on the resource pack list (but not below MC itself or fabric), it gets rid of this issue. Plus, OptiFine's implementation still works for other resource packs. Its a win - win. 

Tested with Fusion 1.0.5a and Optifine HD U I5, Optifabric 1.13.25, and FAPI 0.83.1 (+10000000000 other mods but you get the point)
